### PR TITLE
Document item.c

### PR DESCRIPTION
**Pull request by lhearachel**

### CODE REVIEW

File: include/item.h [hunk 1]
```diff
@@ -75,27 +75,266 @@ typedef struct ItemData {
...
+/**
+ * @brief Get an ID of the specified type for an item.
+ *
+ * Each type points to a different value for the item's entry in sItemArchiveIDs.
+ *
+ * @param item
+ * @param type      The type of ID to get. 
+ * @return The item's ID of the specified type.
+ */
+u16 Item_ID(u16 item, enum ItemFileType type);
```
Comment (Nomura-RH) [inline include/item.h:168]: I feel like this function should be called something more descriptive, but I'm having a hard time coming up with a better name. Maybe `Item_GetIDInItemFile`?

**Review by Nomura-RH — COMMENTED** [review]
Aside from one minor comment about a function name, LGTM

File: include/item.h (hunk 1 above)
Comment (lhearachel) [reply include/item.h:168]: How about `Item_FileID`? All the elements accessed here are pointers to a file in a NARC somewhere, with the exception of the GBA IDs.

Comment (Nomura-RH) [reply include/item.h:168]: Sounds good to me

**Pull request merged**